internal/pkg/enccid/enc_bigint.go

```diff
@@ -0,0 +1,31 @@
+package enccid
+
+import (
+	"math/big"
+)
+
+// EncBigInt is an encodable big integer
+type EncBigInt struct {
```

Comment:
we use bigints in a lot of other types. I would move this to `types` so it can be reused. 
I would also drop the `Enc*` in the name.

Reply:
Removing in favor of shared type